gammapy/modeling/tests/test_parameter.py

```diff
@@ -186,3 +184,16 @@ def test_parameters_autoscale():
     pars.autoscale()
     assert_allclose(pars[0].factor, 2)
     assert_allclose(pars[0].scale, 10)
+
+
+def test_subcovar():
+    pars_0 = Parameters([Parameter("a", 10), Parameter("b", 20), Parameter("c", 30)])
+    pars_1 = Parameters(list(pars_0.unique_parameters)[:2])
```

Comment:
What are you doing here?
`pars_0` doesn't have the same `Parameter` object twice.
Why are you calling `unique_parameters` on it?
Can you make the test case clearer?